src/StoreApi/docs/cart.md

```diff
@@ -339,6 +335,55 @@ Returns the full [Cart Response](#cart-response) on success, or an [Error Respon
 
 If you want to add supplemental cart item data before it is passed into `CartController::add_to_cart` use the [`woocommerce_store_api_add_to_cart_data`](https://github.com/woocommerce/woocommerce-blocks/blob/4d1c295a2bace9a4f6397cfd5469db31083d477a/docs/third-party-developers/extensibility/hooks/filters.md#woocommerce_store_api_add_to_cart_data) filter.
 
+## Add Items
+
+If you want to add multiple items at once, you need to use the batch endpoint:
+
+```http
+POST /wc/store/v1/batch
+```
+
+The JSON payload for adding multiple items to the cart would look like this:
+
+```json
+{
+	"requests": [
+		{
+			"path": "/wc/store/v1/cart/add-item",
+			"method": "POST",
+			"data": {
+				"id": 26,
+				"quantity": 1
+			},
+			"cache": "no-store",
+			"body": {
+				"id": 26,
+				"quantity": 1
+			},
```

Comment:
You don't need the data part, body is all what you need. You also can't post alone with data, so it's mostly useless.

Reply:
I removed the `data` part accordingly.